src/applications/discharge-wizard/components/InstructionsPage.jsx

```diff
@@ -106,7 +106,7 @@ class InstructionsPage extends React.Component {
                               itemProp="acceptedAnswer"
                               itemScope
                               itemType="http://schema.org/Answer"
-                              aria-hidden={!this.state.q1}
+                              aria-hidden={!this.state['other-options']}
```

Comment:
This wasn't working correctly after removing uswds. `this.state.q1` doesn't have an initial value, so this wasn't correct. I think it was working due to the uswds javascript. formation.js doesn't add an event listener to the element if it's a React component or has a previous event listener assigned to it.

Reply:
Sounds like we need to test out the accordions in any React apps that have them.